utils/fetchEarthquakes.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
```

Comment:
THis whole module can probably be renamed... but it's probably not too important if I'm adding a db in the next few days